content/learn/book/ecs/entities-components/_index.md

```diff
@@ -6,6 +6,307 @@ page_template = "book-section.html"
+Components store this data, and define the overlapping categories that the entity belongs to.
 
-TODO: show how to create an entity, add components to it, and query for it in pure `bevy_ecs`
+Informally, we use the term "entity" to refer to the conceptual entry in our [`World`]: all of the component data with the correct identifier, although it's very rare to use all of the data for a single entity at once.
+If you're an experienced programmer, you can reason about the [`World`] as something like a (very fast) [`HashMap`] from [`Entity`] to a collection of components.
+
+[`Entity`]: https://docs.rs/bevy/latest/bevy/ecs/entity/struct.Entity.html
+[`HashMap`]: https://doc.rust-lang.org/std/collections/struct.HashMap.html
+[`World`]: https://docs.rs/bevy/latest/bevy/ecs/world/struct.World.html
+
+## Spawning and despawning entities
+
+Before you can do much of anything in Bevy, you'll need to **spawn** your first entity, adding it to the app's [`World`].
+Once entities exist, they can likewise be despawned, deleting all of the data stored in their components and removing it from the world.
+
+Spawning and despawning entities can have far-reaching effects, and so cannot be done immediately (unless you are using an [exclusive system](../exclusive-world-access/_index.md)).
+As a result, we must use [`Commands`], which queue up work to do later.
+
+```rust
+// The `Commands` system parameter allows us to generate commands
+// which operate on the `World` once all of the current systems have finished running
+fn spawning_system(mut commands: Commands){
+    // Spawning a single entity with no components
+    commands.spawn();
+    // Getting the `Entity` identifier of a new entity
+    let my_entity = commands.spawn().id();
+    // Selecting and then despawning the just-spawned second entity
+    commands.entity(my_entity).despawn();
+}
+```
+
+[`Commands`]: https://docs.rs/bevy/latest/bevy/ecs/system/struct.Commands.html
+
+## Working with components
+
+Entities are entirely bare when they're spawned: they contain no data other than their unique [`Entity`] identifier.
+This of course is not very useful, so let's discuss how we can add and remove components to them which store data and enable behavior through systems.
+
+[`Entity`]: https://docs.rs/bevy/latest/bevy/ecs/entity/struct.Entity.html
+
+### Defining components
+
+To define a component type, we simply implement the [`Component`] [trait](https://doc.rust-lang.org/book/ch10-02-traits.html) for a Rust type of our choice.
+You will almost always want to use the `#[derive(Component)]` [macro](https://doc.rust-lang.org/reference/attributes/derive.html) to do this for you; which quickly and reliably generates the correct trait implementation.
+Any underlying component data must be `Send + Sync + 'static` (enforced by the [trait bounds](https://doc.rust-lang.org/book/ch10-02-traits.html#trait-bound-syntax) on [`Component`]).
```

Comment:
Soon, this won't be true anymore.

- bevyengine/bevy#4680

This is especially true if the book gets published after 0.8, for which the PR is probably going to be already merged in.

Reply:
Yeah, we should cut this bit from the book intro and have a section about the trait bounds on the Component docs IMO.